src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -114,8 +114,15 @@ static unique_ptr<RadialGradient> _applyRadialGradientProperty(SvgStyleGradient*
         g->radial->fy = g->radial->fy * rh + ry;
     }
 
-    //TODO: Radial gradient transformation is not yet supported.
-    //if (g->transform) {}
+    //TODO: Radial gradient transformation - all tests possible after rx/ry implementation
+    if (g->transform) {
+        auto cx = g->radial->cx * g->transform->e11 + g->radial->cy * g->transform->e12 + g->transform->e13;
+        g->radial->cy = g->radial->cx * g->transform->e21 + g->radial->cy * g->transform->e22 + g->transform->e23;
+        g->radial->cx = cx;
+
+        auto sx = sqrt(pow(g->transform->e11, 2) + pow(g->transform->e21, 2));
```

Comment:
Can you explain this calculation?
Why g->transform->e21 ?

Reply:
for now it doesn't make any difference, we can use here e21 or e12.
it is because tvg doesn't support radial grad radius with respect to the shape width/height (#37)

after we solve #37 I think we should add one more step so we will be able to determine what was applied first - scaling or rotation, so we can properly find the  sx and sy values.